packages/roosterjs/lib/createEditor.ts

```diff
@@ -17,6 +18,7 @@ export default function createEditor(
         new HyperLink(),
         new PasteManager(),
         new TabIndent(),
+        new BlockQuote(),
```

Comment:
Not sure why BlockQuoate is in the default plugin list.

Reply:
Removed it as we moved the logic in TabIndent.